pom.xml

```diff
@@ -14,8 +14,8 @@
   </licenses>
 
   <properties>
-    <raml-module-builder.version>29.0.1</raml-module-builder.version>
-    <vertx.version>3.8.1</vertx.version>
+    <raml-module-builder.version>30.0.1</raml-module-builder.version>
```

Comment:
Why not 30.0.2? Or was it unavailable back then?

Reply:
This PR had been prepared before the 30.0.2 was released.